### PR TITLE
supports windows msvc100 build.

### PR DESCRIPTION
**Pull request by cvvergara**
pull request to discuss
Code differences

@pgRouting/admins

### CODE REVIEW

File: src/apsp_johnson/src/apsp_johnson.c [hunk 1]
```diff
@@ -32,17 +32,26 @@
 
 //-------------------------------------------------------------------------
 
-Datum apsp_johnson(PG_FUNCTION_ARGS);
+PGDLLEXPORT Datum apsp_johnson(PG_FUNCTION_ARGS);
```
Comment (cvvergara) [inline src/apsp_johnson/src/apsp_johnson.c]: What would be the wrapper to this?
```
#ifdef ¿?
PGDLLEXPORT 
#endif

**Comment by sanak** [comment]
@cvvergara 
In my memory, my pgrouting4w master-msvc branch is newer than develop-msvc branch.

File: src/apsp_johnson/src/apsp_johnson.c (hunk 1 above)
Comment (cvvergara) [inline src/apsp_johnson/src/apsp_johnson.c]:
```
#ifdef _MSC_VER
PGDLLEXPORT 
#endif
```
will that work?

**Comment by cvvergara** [comment]
ok, I will use the other one.

**Pull request closed without merging**